test.js

```diff
@@ -0,0 +1,6 @@
+console.log('foo');
+console.log('foo')
+    console.log('foo')
+    function a(foo) {
```

Comment:
__[development][eslint]__
* Expected indentation of 0 spaces but found 4. (indent)
* 'a' is defined but never used. (no-unused-vars)
* Missing space before function parentheses. (space-before-function-paren)

<!-- Comment by [SideCI](https://sideci.com) -->